homeassistant/components/lametric/quality_scale.yaml

```diff
@@ -0,0 +1,75 @@
+rules:
```

Comment:
General note, why do we store the hass config in `hass.data[DOMAIN]`?

Reply:
Old notify platform, we need to pass along the config.

Reply:
But I haven't seen any usage of `hass.data[DOMAIN]["hass_config"]` or did I miss that

Reply:
![CleanShot 2024-12-13 at 10 40 21](https://github.com/user-attachments/assets/fb02ca54-c008-4951-95bf-28228f985bd8)

Reply:
Oh I missed that one whoops